SampleApp/openwrap/src/main/java/com/pubmatic/openwrap/POWAdRequest.java

```diff
@@ -35,60 +36,81 @@ public class POWAdRequest {
+    private static final String METRO_KEY = "dev.geo.metro";
+    private static final String ZIP_KEY = "dev.geo.zip";
+    private static final String UTC_OFFSET_KEY = "dev.geo.utcoffset";
+    private static final String MAKE_KEY = "dev.make";
+    private static final String MODEL_KEY = "dev.model";
+    private static final String OS_KEY = "dev.os";
+    private static final String OSV_KEY = "dev.osv";
+
     private static final String BIDDER_PARAM_KEY = "pwtbidrprm";
     // User Object
-    private static final String BIRTH_YEAR_KEY = "pwtyob";
-    private static final String GENDER_KEY = "pwtgender";
+    private static final String BIRTH_YEAR_KEY = "user.yob";
+    private static final String GENDER_KEY = "user.gender";
     // App Object
-    private static final String APP_KEY = "pwtapp";
-    private static final String APP_NAME_KEY = "pwtappname";
-    private static final String APP_BUNDLE_KEY = "pwtappbdl";
-    private static final String APP_DOMAIN_KEY = "pwtappdom";
-    private static final String APP_URL_KEY = "pwtm_url";
-    private static final String APP_STORE_URL_KEY = "pwtappurl";
-    private static final String APP_CATEGORY_KEY = "pwtappcat";
-    private static final String APP_PAID_KEY = "pwtapppd";
+    private static final String APP_KEY = "app.id";
+    private static final String APP_NAME_KEY = "app.name";
+    private static final String APP_BUNDLE_KEY = "app.bundle";
+    private static final String APP_DOMAIN_KEY = "app.domain";
+    private static final String APP_URL_KEY = "app.url";
+    private static final String APP_STORE_URL_KEY = "app.storeurl";
+    private static final String APP_CATEGORY_KEY = "app.cat";
+    private static final String APP_PAID_KEY = "app.paid";
     // Extension param
-    private static final String GDPR_KEY = "pwtgdpr";
-    private static final String GDPR_CONSENT_KEY = "pwtcnst";
-    private static final String CCPA_KEY = "pwtccpa";
+    private static final String GDPR_KEY = "regs.ext.gdpr";
+    private static final String GDPR_CONSENT_KEY = "user.consent";
+    private static final String CCPA_KEY = "regs.ext.us_privacy";
+    //AdPod param
+
+    private static final String MINADS = "imp.vid.ext.adpod.minads";
+    private static final String AD_MINDURATION = "imp.vid.ext.adpod.adminduration";
+    private static final String MAXADS = "imp.vid.ext.adpod.maxads";
+    private static final String AD_MAXDURATION = "imp.vid.ext.adpod.admaxduration";
+
     // Default values
     private static final String APP_PARAM_VALUE = "1";
     private static final String AD_FORMAT_VALUE = "video";
     private static final String AD_SERVER_VALUE = "DFP";
     private static final String RESPONSE_FORMAT_VALUE = "json";
-    private static final String REQUEST_MIME_VALUE = "1";
-
-    // OpenWrap Base Url
-    private static final String OW_URL = "https://ow.pubmatic.com/openrtb/2.5/video";
+    private static final String REQUEST_MIME_VALUE = "video/mp4";
```

Comment:
Instead of changing the existing request builder, you can have separate request builder for `https://ow.pubmatic.com/video/json` 
You can use factory pattern to have the use of specific implementation.

Reply:
We know the old API will not be used, should we keep both apis and increase code size?